timesketch/lib/emojis.py

```diff
@@ -42,7 +42,265 @@
     'SLEEPING_FACE': emoji('&#x1F634', 'Activity outside of regular hours'),
     'SPARKLES': emoji('&#x2728', 'New entity created'),
     'UNLOCK': emoji('&#x1F513', 'Logoff activity'),
-    'WASTEBASKET': emoji('&#x1F5D1', 'Deletion activity')
+    'WASTEBASKET': emoji('&#x1F5D1', 'Deletion activity'),
+    'AC': emoji('&#x1F1E6&#x1F1E8', 'Ascension Island'),
```

Comment:
Same here, Can prepend these with something like FLAG_ ?

Reply:
Sure can, thanks for the suggestion.  Updated.